app/models/invenio_rdm_record_converter.rb

```diff
@@ -626,4 +626,16 @@ def sizes
   def user_email(user)
     user.email == "joshelder@northwestern.edu" ? "JoshElder@northwestern.edu" : user.email
   end
+
+  # This is the current list of complex headers that were not mapping correctly. They will be exported as keywords
+  def unmappable_complex_headers
```

Comment:
I dont love hardcoding this, but as I've been saying across this PR - it gets the job done.

Reply:
There is no rule for defining those, eh? It will indeed have to do :shrug: